spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -951,6 +951,32 @@ def a; end
       YAML
     end
 
+    context 'when configurable parameter is an obsolete parameter' do
+      it 'displayed as a new parameter setting value without duplication' do
+        create_file('.rubocop.yml', <<~YAML)
+          AllCops:
+            NewCops: enable
+          Naming/MethodName:
+            IgnoredPatterns:
```

Comment:
IMO, inserting comment information like the following might make this test a little more maintainable :-)

```suggestion
            # `IgnoredPatterns` is obsolete. `AllowedPatterns` is newly used.
            IgnoredPatterns:
```

Reply:
Looks great! Added it.